cassava.cabal

```diff
@@ -83,9 +83,9 @@ Library
     Data.Csv.Incremental
     Data.Csv.Parser
     Data.Csv.Streaming
+    Data.Csv.Conversion
```

Comment:
Fyi, this is the reason I'm hesitant about this change, as it opens up something to the public that was considered internal before. I'll have to think about this, sorry this is taking so long.

Reply:
I perhaps instead of all of conversion we could just move Parser somewhere else? It looks like that's all we're using here.

Reply:
would it suffice for you to have access to the opaque `Parser` and `runParser`? or do you need more?